pkg/admission/validator/shoot.go

```diff
@@ -127,11 +123,6 @@ func (s *shoot) validateShoot(_ context.Context, shoot *core.Shoot) error {
 		return err
 	}
 
-	csiMigrationVersion, err := semver.NewVersion(aws.GetCSIMigrationKubernetesVersion(&extensionscontroller.Cluster{Shoot: shootV1beta1}))
```

Comment:
provider-aws is supporting the `alpha.csimigration.shoot.extensions.gardener.cloud/kubernetes-version` annotation for delaying the CSI migration. Before cleaning up the CSI related handling did we checked whether this feature is used on Shoots on our landscapes? You could add at least a release not of type breaking that support for `alpha.csimigration.shoot.extensions.gardener.cloud/kubernetes-version` is being removed and provider-aws will assume that the CSI migration completed in 1.18.

Reply:
Checked for our landscapes. The annotation `alpha.csimigration.shoot.extensions.gardener.cloud/kubernetes-version`  is not used for any shoot.